pkg/network/ebpf/c/runtime/usm.c

```diff
@@ -262,10 +262,13 @@ int uprobe__crypto_tls_Conn_Read__return(struct pt_regs *ctx) {
         return 0;
     }
 
-    char *buffer_ptr = (char*)call_data_ptr->b_data;
-    bpf_map_delete_elem(&go_tls_read_args, (go_tls_function_args_key_t*)&call_key);
+    conn_tuple_t copy = *t;
+    flip_tuple(&copy);
+
+    char *buffer_ptr = (char *)call_data_ptr->b_data;
+    bpf_map_delete_elem(&go_tls_read_args, (go_tls_function_args_key_t *)&call_key);
 
-    tls_process(ctx, t, buffer_ptr, bytes_read, GO);
+    tls_process(ctx, &copy, buffer_ptr, bytes_read, GO);
```

Comment:
please document, it is unclear for the naive reader

Reply:
also, try to limit the change, by moving the creation of `copy` to be just on top of `tls_process`